python/test/unit/fem/test_bcs.py

```diff
@@ -43,3 +45,47 @@ def test_locate_dofs_geometrical():
         # Check correct dof returned in V
         coords_V = V.tabulate_dof_coordinates()
         assert np.isclose(coords_V[dofs[0][1]], [0, 0, 0]).all()
+
+
+def test_overlapping_bcs():
```

Comment:
Should this test also perform a solve and check that the solution matches the boundary condition?

Reply:
The solve shouldn't affect the boundary condition if the value in the rhs is correct and the corresponding diagonal entry in the lhs is 1.